config/environments/test.rb

```diff
@@ -8,13 +8,13 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
-  config.cache_classes = false
-  config.action_view.cache_template_loading = true
+  # Turn false under Spring and add config.action_view.cache_template_loading = true.
+  config.cache_classes = true
```

Comment:
I thought this config was deprecated in favor of `.enable_reloading` 🤔 https://edgeguides.rubyonrails.org/configuring.html#config-cache-classes

Reply:
https://github.com/rootstrap/rails_hotwire_base/pull/43/files#r991410060